tests/scripts/findkernelversion.sh

```diff
@@ -17,8 +17,31 @@ chmod a+x bin/regctl
 export PATH=$(pwd)/bin:${PATH}
 
 # calculate kernel version of latest image
-regctl image get-file ghcr.io/nvidia/driver:base-${BASE_TARGET}-${KERNEL_FLAVOR}-${DRIVER_BRANCH} /var/kernel_version.txt ./kernel_version.txt
-export $(grep -oP 'KERNEL_VERSION=[^ ]+' ./kernel_version.txt)
+regctl image get-file ghcr.io/nvidia/driver:base-${BASE_TARGET}-${KERNEL_FLAVOR}-${DRIVER_BRANCH} /var/kernel_version.txt ./kernel_version.txt 2>/dev/null || true
+if [[ -f ./kernel_version.txt && -s ./kernel_version.txt ]]; then
```

Comment:
As discussed offline, let's see if we can remove this conditional logic as we are building the base image in a previous step in this workflow.

Reply:
this require pushing of base image. 

TODO: follow up added